### ISSUE TITLE
AttributeError from import collision on generated dataclasses with similar directory names

### ISSUE THREAD

**Issue by v1b1** [danielgtaylor/python-betterproto#190]
**Trigger Conditions**
1) Use Python 3.8 and current version of `betterproto` module
2) Store multiple `.proto` files deep in directories with the same immediate parent name (e.g. `foo/v1/*.proto` and `bar/v1/*.proto`)
3) Create a `target/foobar.proto` file that imports from both packages:
```protobuf
syntax = "proto3";
import "foo/v1/foo.proto";
import "bar/v1/bar.proto";
...
```
4) Run the following command to generate the dataclasses:
```bash
python -m grpc_tools.protoc --python_betterproto_out=out \
    -I ./project \
    ./project/target/*.proto
```

**Effects**
The resultant `target.py`:
```python
from .foo import v1
from .bar import v1
...
@dataclass
class Foobar(betterproto.Message):
    foo_type: v1.Foo = betterproto.message_field(1)
    ...
```
Causes the following exception when imported:
```
AttributeError: module 'project.bar.v1' has no attribute 'Foo'
```

**Comment by Gobot1234** [comment]
When you say current version of betterproto is this 1.2.5, 2.0.0b2 or github master?

**Comment by v1b1** [comment]
This occurs with the 1.2.5 release.

**Comment by Gobot1234** [comment]
Try GitHub master

**Comment by v1b1** [comment]
Thanks, that does seem to resolve the issue on my local machine. Is there a timeline for when this will appear in a release? I'm stuck with a build pipeline at my org that will not allow pip install directly from github.

**Comment by Gobot1234** [comment]
It should be fixed on 2.0.0b2 which is installable from pypi using pip install betterproto=2.0.0b2

**Comment by v1b1** [comment]
Thanks, closing this issue.
